### PR TITLE
Fix browser tests on Travis

### LINKED ISSUE #168: Fix Travis
[Issuehunt badges](https://issuehunt.io/r/sindresorhus/ky/issues/168)

https://travis-ci.com/sindresorhus/ky/jobs/227726270

Travis is slow. I think we might just need to increase the Puppeteer timeout somehow.

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Backers (Total: $40.00)

- [issuehunt issuehunt](https://issuehunt.io/u/issuehunt) ($40.00)
### Submitted pull Requests
- [#177 Fix Travis CI - Serialize Browser Test](https://issuehunt.io/r/sindresorhus/ky/pull/177)
---

### Tips

- Checkout the [Issuehunt explorer](https://issuehunt.io/r/sindresorhus/ky/) to discover more funded issues.
- Need some help from other developers? [Add your repositories](https://issuehunt.io/r/new) on IssueHunt to raise funds.
---
IssueHunt has been backed by the following sponsors. [Become a sponsor](https://issuehunt.io/membership/members)
</details>

### PR DESCRIPTION
**Pull request by dirathea**
Fixes #168 

# Summary
Fix travis config

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Referenced issues

This pull request has been submitted to:
- [#168: Fix Travis](https://issuehunt.io/repos/147375544/issues/168)
---

IssueHunt has been backed by the following sponsors. [Become a sponsor](https://issuehunt.io/membership/members)
</details>

### CODE REVIEW

**Comment by sholladay** [comment]
Thanks for giving this a shot!

At the moment, this PR has the same CI failure as previous builds. I think we should try limiting the concurrency so that there are fewer browser windows opened simultaneously.

**Comment by dirathea** [comment]
Ah! Nice suggestions @sholladay Ok, I'll try

**Comment by szmarczak** [comment]
@sholladay What about running all the browser tests serially?

**Comment by dirathea** [comment]
I think that's good idea for specify browser test as serial, while others could run as concurrent. I think that way will speed up test time. I'll try

**Comment by szmarczak** [comment]
> I think that way will speed up test time.

Not necessarily, but it will guarantee that the tests will pass. It may be quite slower, but that won't be much since there aren't so many browser tests here.

**Comment by szmarczak** [comment]
Yup, ~10s slower.

File: test/browser.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import test from 'ava';
 import createTestServer from 'create-test-server';
```
Comment (sindresorhus) [inline test/browser.js]: You can just do `import {serial as test} from 'ava';` to make all of them serial instead of at each callsite.

**Pull request merged**